src/bottomSheet/BottomSheet.js

```diff
@@ -86,14 +65,16 @@ const styles = StyleSheet.create({
 
 BottomSheet.defaultProps = {
   list: [],
-  buttonProps: {},
-  cancelButtonIndex: null,
+  modalProps: {},
+  onClose: () => {},
+  isVisible: false,
 };
 
 BottomSheet.propTypes = {
   list: PropTypes.array,
-  cancelButtonIndex: PropTypes.number,
-  buttonProps: PropTypes.object,
+  modalProps: PropTypes.object,
+  onClose: PropTypes.func,
```

Comment:
If `onClose` is already provided by `Modal` then I don't think it has to be included here.